sampler/stories/default/Heading.js

```diff
@@ -0,0 +1,32 @@
+import Heading, {HeadingBase} from '@enact/agate/Heading';
+
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import {boolean, select, text} from '../../src/enact-knobs';
+import {mergeComponentMetadata} from '../../src/utils';
+
+Heading.displayName = 'Heading';
+const Config = mergeComponentMetadata('Heading', Heading, HeadingBase);
+
+const prop = {
+	colors: ['', '#E6444B', '#FDC902', '#986AAD', '#4E75E1', '#30CC83', '#44C8D5', '#47439B', '#2D32A6', '#4E75E1'],
```

Comment:
The colors, they do nothing!

(or I don't understand what it's doing)

Reply:
It only has a noticeable effect on the 'title' sized heading at this time. The other heading sizes will likely have some visual usage of this color in Gallium after the designs settle.

Reply:
It might be busted?

Default:
<img width="529" alt="Screen Shot 2019-09-05 at 1 57 14 PM" src="https://user-images.githubusercontent.com/192353/64371027-3a0e8000-cfe5-11e9-945e-ad5128991380.png">

Custom color:
<img width="529" alt="Screen Shot 2019-09-05 at 1 57 44 PM" src="https://user-images.githubusercontent.com/192353/64371048-4561ab80-cfe5-11e9-956a-64a4cdfabfb9.png">

Reply:
The prop name is different now (in the latest code), it's just `color`, so that may be a factor. The layering looks odd, like the text is translucent, when it shouldn't be...

Reply:
I updated the sample to use the new prop name `color`.